### ISSUE TITLE
Translation for infinity

### ISSUE THREAD

**Issue by ekrebs5** [hpi-swt2/workshop-portal#378]
If a workshop can have an infinite number of participants the number is shown as "infinity".
This should be translated in views, at the moment eg we have
"Maximale Teilnehmerzahl: Infinity"
on the event page.

**Comment by bjrne** [comment]
Since my team started with the event page, we discussed early-on whether the "infinite"-number of participants feature is necessary and decided against it. There is **no** real world application where an *unlimited* number of participants is wanted. Our PO agreed to this. We decided against it, because it also introduces a lot of unnecessary complexity everywhere. Visual cornercases, weird database logic and unnecessary tests...

Before implementing all of these cases I'd like to have the POs opinion on this and whether the client really wants this, since this means a lot of effort, not only writing this but also maintaining everything.  @hpi-swt2/product-owner-2016 
@hpi-swt2/scrum-master-2016
